packages/gatsby-theme-api-docs/src/hooks/use-apis.js

```diff
@@ -26,4 +26,10 @@ const useApis = () => {
   return queryResult.allRamlApi.nodes;
 };
 
-export default useApis;
+export const useApiByKey = (apiKey) => {
```

Comment:
refactored this hook to be structured like the other main hooks which expose helper functions and the actual hook together from one file, too.